### ISSUE TITLE
Add EntityFacts to describedBy object

### ISSUE THREAD

**Issue by acka47** [hbz/lobid-gnd#294]
@felixlohmeier requested via email a way to query `describedBy.dateModified` and to also get the last modification date of data from EntityFacts. One option would be 

1. to make `describedBy` an array (which would be an API break) and 
2. to inject an object into `describedBy` for each resource that gets information from EntityFacts with `dateModified` of the last day we updated from EntityFacts.

E.g. for https://lobid.org/gnd/124767079:

```json
{
  "describedBy":[
    {
      "id":"https://d-nb.info/gnd/124767079/about",
      "license":{
        "id":"http://creativecommons.org/publicdomain/zero/1.0/",
        "label":"http://creativecommons.org/publicdomain/zero/1.0/"
      },
      "dateModified":"2020-12-18T16:08:46.000"
    },
    {
      "id":"https://hub.culturegraph.org/entityfacts/124767079",
      "license":{
        "id":"http://creativecommons.org/publicdomain/zero/1.0/",
        "label":"http://creativecommons.org/publicdomain/zero/1.0/"
      },
      "dateModified":"2021-08-27"
    }
  ]
}
```  

To clarify, we would update the `dateModified` with each update form EntityFacts although for most resources the EntityFacts information doesn't change between updates. @felixlohmeier, would this suit your use case?

**Comment by felixlohmeier** [comment]
Yes, that would solve my use case.I am looking for a way to pre-filter weekly bulk downloads to save resources.
